scripts/process.zs

```diff
@@ -276,6 +276,12 @@ function beneficiate(
     crush(input, dustOrGem, exceptions ~ 'macerator thermalCentrifuge crushingBlock', extraList, extraChances, { bonusType: 'MULTIPLY_OUTPUT' });
   }
 
+  // Mass Spectrometer Dirty Gem processing
+  val gem = utils.getSomething(oreName, ['gem', 'any'], amount + step);
```

Comment:
`['gem', 'any']` means that utils will try to find gems for ore output and if there is no gem form (like ingots). But, isnt dusts is prefferable? MassSpectrometr in your original design output dusts first. You can change it to `['gem', 'dust', 'any']` or use already defined variable `dustOrGem`.